### DISCUSSION TITLE
Link to downlod rsync is broken

### DISCUSSION THREAD

**Discussion by elexus** [firecow/gitlab-ci-local#1399] [General]
Exact link in your project is ok, but link on that page to dowload rsync is broken. Maybe you could provide direct link to download it?

**Comment by elexus** [comment]
Here is a link to comment, that resoves the problem:
https://gist.github.com/radleta/0b337a2b14f761951cf2aab0578512b9?permalink_comment_id=4442198#gistcomment-4442198

**Comment by firecow** [comment]
Very nice @elexus I've added it to the readme https://github.com/firecow/gitlab-ci-local/pull/1400
